src/fragments/lib/datastore/flutter/data-access/observe-update-snippet.mdx

```diff
@@ -0,0 +1,55 @@
+```dart
+void main() {
+  runApp(MyApp());
+}
```

Comment:
If we do not care about MaterialApp and any other elements, why do we need this main function in the first place?